mobile-app/app/components/TransactionCard.tsx

```diff
@@ -0,0 +1,118 @@
+import { ThemedViewV2, ThemedTextV2, ThemedTouchableOpacityV2 } from '@components/themed'
+import BigNumber from 'bignumber.js'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+
+interface TransactionCardProps {
+  maxValue: BigNumber
+  onChange: (amount: string, type: AmountButtonTypes) => void
+  status?: string
+}
+
+export enum AmountButtonTypes {
+  twentyFive = '25%',
+  half = '50%',
+  seventyFive = '75%',
+  max = 'MAX'
+}
+
+export function TransactionCard ({ maxValue, onChange, status, children }: React.PropsWithChildren<TransactionCardProps>): JSX.Element {
+  return (
+    <ThemedViewV2
+      light={tailwind('bg-mono-light-v2-00', {
+        'border-0.5 border-mono-light-v2-800': status === 'active',
+        'border-0.5 border-red-v2': status === 'error'
+      })}
+      dark={tailwind('bg-mono-dark-v2-00', {
+        'border-0.5 border-mono-dark-v2-800': status === 'active',
+        'border-0.5 border-red-v2': status === 'error'
+      })}
+      style={tailwind('rounded-lg-v2', {
+        'border-0.5 border-red-v2': status === 'error'
+      })}
+    >
+      <ThemedViewV2
+        light={tailwind('bg-mono-light-v2-00')}
+        dark={tailwind('bg-mono-dark-v2-00')}
+        style={tailwind('pl-5 pr-5 pt-2 mr-px rounded-t-lg-v2')}
+      >
+        {children}
+      </ThemedViewV2>
+      <ThemedViewV2
+        light={tailwind('border-mono-light-v2-300')}
+        dark={tailwind('border-mono-dark-v2-300')}
+        style={tailwind('flex flex-row justify-around items-center pt-2 pb-4 border-t-0.5')}
```

Comment:
```suggestion
        style={tailwind('flex flex-row justify-around items-center pb-4')}
```
send has no border, can we move this on `{children}` bottom border too?